mcs/class/Mono.Debugger.Soft/Test/dtest-app.cs

```diff
@@ -581,11 +583,27 @@ public static void ss7_3 ()
 		ss_nested_with_two_args(ss_nested_arg (), ss_nested_arg ());
 	}
 
+	[MethodImplAttribute (MethodImplOptions.NoInlining)]
+	public static void ss_nested_with_three_args_wrapper () {
+		ss_nested_with_three_args(ss_nested_arg (), ss_nested_arg (), ss_nested_arg ());
```

Comment:
is there any order expected? should we rename it rather to `ss_nested_arg1`, `ss_nested_arg2` etc. and verify that in the test?

Reply:
Yes, yes, good idea, I will change the names!

Reply:
Done!